### PR TITLE
chore(modheader): lower store cache size for LN

### PR DESCRIPTION
**Pull request by Wondertan**
Part of an effort to decrease the memory footprint for LNs. Headers are pretty beefy.

Extracted from #2960

### CODE REVIEW

File: nodebuilder/header/config.go [hunk 1]
```diff
@@ -43,13 +43,19 @@ func DefaultConfig(tp node.Type) Config {
 		Store:        store.DefaultParameters(),
 		Syncer:       sync.DefaultParameters(),
 		Server:       p2p_exchange.DefaultServerParameters(),
+		Client:       p2p_exchange.DefaultClientParameters(),
 	}
 
 	switch tp {
 	case node.Bridge:
+		cfg.Client = p2p_exchange.ClientParameters{}
```
Comment (renaynay) [inline nodebuilder/header/config.go:51]: why is this necessary?

Comment (Wondertan) [reply nodebuilder/header/config.go:51]: I just dumbly followed the behavior that was before, where Bridge had empty Client params. But now I think it is fine to remove it.

Comment (Wondertan) [reply nodebuilder/header/config.go:51]: Ah wait. It may actually impact the `celestia init`. Please check on the BN if the client config is initialized there.

Comment (Wondertan) [reply nodebuilder/header/config.go:51]: @renaynay

Comment (renaynay) [reply nodebuilder/header/config.go:51]:
```
	// we do not create a client for bridge nodes
	if tp == node.Bridge {
		return nil
	}
```

so no

**Review by renaynay — APPROVED** [review]

**Review by vgonkivs — APPROVED** [review]
